src/comm/socket.cc

```diff
@@ -120,7 +121,7 @@ int Router::Bind(const std::string& endpoint) {
     port = zsock_bind(router_, "%s", endpoint.c_str());
   }
   CHECK_NE(port, -1) << endpoint;
-  LOG(INFO) << "bind successfully to " << zsock_endpoint(router_);
+  LOG_IF(INFO, enable_info_log) << "bind successfully to " << zsock_endpoint(router_);
```

Comment:
We were not consistent with the Log msg format.
Maybe uniform them to start with a capital word, e.g., "Bind..."